config/api/src/main/java/org/jboss/windup/config/PreRulesetEvaluation.java

```diff
@@ -0,0 +1,16 @@
+package org.jboss.windup.config;
+
+import org.ocpsoft.rewrite.config.Rule;
+
+/**
+ * Denotes an element that takes some action before the execution of the {@link Rule} pipeline.
```

Comment:
Isn't this only valid for Rule objects? It might be good to make that clear (if that is the case).